internal/model/game.go

```diff
@@ -1,24 +1,28 @@
 package model
 
+type Game struct {
+	score gameScore
+}
+
 type gameScore struct {
 	BlueScore  int `json:"blueScore"`
 	WhiteScore int `json:"whiteScore"`
 }
 
-func New() *gameScore {
-	return &gameScore{}
+func New() *Game {
```

Comment:
What is the reason for changing the private struct to the public one?

Reply:
We didn't change this structure. We've made new public structure "Game" that contains a private structure as a field. We've made "Game" structure public to be able to declare its instances in other packages (for example in server package).

Reply:
The previous developer added the `New(...) ...` method to allow struct to be declared in another package. Right now users in the `server` package may declare the `Game` struct in two ways:

```
v1 := model.Game{}

or

v2 := model.New()
```

the result is exactly the same. For me, it looks like a possible confusion we can avoid